src/Fantomas/Trivia.fs

```diff
@@ -282,7 +282,25 @@ let private addTriviaToTriviaNode
 
     | { Item = Keyword({ Content = keyword} as kw); Range = range } when (keyword = "override" || keyword = "default" || keyword = "member" || keyword = "abstract") ->
         findMemberDefnMemberNodeOnLine triviaNodes range.StartLine
-        |> updateTriviaNode (fun tn -> tn.ContentItself <- Some (Keyword(kw))) triviaNodes
+        |> updateTriviaNode (fun tn ->
+            match tn.Type, tn.ContentItself with
+            | MainNode ("SynMemberSig.Member"), Some (Keyword ({ Content = existingKeywordContent } as token)) when existingKeywordContent =
```

Comment:
Oh my :P the actual content of the code looks fine to me, and I know it's entirely not my place to comment on code layout in a codebase I barely know, but a 120-space indent seems a bit much to me!

Reply:
Yeah, I know, but uh the thing is that I formatted it with Fantomas, so yeah 🙈.
I guess we can later use it as sample to improve.